packages/lab/stories/badge/badge.doc.stories.mdx

```diff
@@ -15,7 +15,7 @@ import {
 
 # Badge
 
-A Badge is a numeric annotation that indicates the number of outstanding items that need to be addressed. It appears on the top-right of an element, usually an icon. The badge can accommodate values of up to four digits in length.
+Badge is an alpha or numeric character annotation indicating the number of outstanding items. It appears on the top-right of an element, usually an icon. 
```

Comment:
Badge can be used with both numeric 'or' alpha characters - do you think this needs a comment about not using them together or should that be obvious ?

Reply:
I think this is fine